extensions/wikia/ContentReview/controllers/templates/JSPagesSpecialController_index.php

```diff
@@ -31,7 +31,7 @@
 				<td><?= $page['latestReviewed']['statusMessage'] ?></td>
 				<td><?= $page['liveRevision']['statusMessage'] ?></td>
 				<td>
-					<?php if ( $page['submit'] ): ?>
+					<?php if ( isset( $page['submit'] ) ): ?>
```

Comment:
I would use !empty($page[''submit']) since $page['submit'] = false would resolve this if to false and isset would return true in that case

Reply:
thx, this'll be more accurate!